docs/node/run-your-node/archive-node.md

```diff
@@ -12,15 +12,16 @@ historic state which is pruned in dump-restore network upgrades.
 
 ## Prerequisites
 
-Archive node requires a pre-existing oasis-node state.
+Running an archive node requires a pre-existing oasis-node state.
 
-## Configuration
+## Configuration (v23 onward)
```

Comment:
Not sure, if we should either stick to the upgrade names or the Oasis Core versions. In any case, the titles should be consistent IMO.

```suggestion
## Configuration (Oasis Core 23 and later)
```